packages/@expo/config-plugins/src/ios/DevelopmentTeam.ts

```diff
@@ -0,0 +1,86 @@
+import type { ExpoConfig } from '@expo/config-types';
+import fs from 'node:fs';
+import xcode, { type XCBuildConfiguration } from 'xcode';
+
+import { getAllPBXProjectPaths } from './Paths';
+import { getNativeTargets } from './Target';
+import { getBuildConfigurationsForListId } from './utils/Xcodeproj';
+import { trimQuotes } from './utils/string';
+import type { ConfigPlugin, XcodeProject } from '../Plugin.types';
+import { withXcodeProject } from '../plugins/ios-plugins';
+
+/**
+ * Set the Apple development team ID for all build configurations using the first native target.
+ */
+export const withDevelopmentTeam: ConfigPlugin<{ developmentTeam?: string } | void> = (
+  config,
+  { developmentTeam } = {}
+) => {
+  return withXcodeProject(config, (config) => {
+    // TODO: maybe infer from `EXPO_APPLE_TEAM_ID` env variable?
```

Comment:
i think @EvanBacon prefers to avoid reading env vars from config plugins. folks can explicitly pass in `process.env.EXPO_APPLE_TEAM_ID` in their app config if they like